design/README.md

```diff
@@ -0,0 +1,52 @@
+# The Design Tools We Use and Why We Use Them
+
+## Sketch for creating comps 
+We use Sketch to create comps for websites. Sketch is particularly good at mocking up UIs because of how it blends vector and bitmap artwork. Comps created in Sketch are used to communicate a design vision to the client and our team, not necessarily to show exactly what a website will look like.
+
+We use a team-based license for Sketch. Talk with Jeremy to set that up. 
```

Comment:
I think we try to say titles and not people, so "Creative Director" instead of Jeremy ?